src/backend/TdConsumer.ts

```diff
@@ -78,26 +87,25 @@ export default class TdConsumer {
         }
     }
 
+    // fetches TD
+    async fetchTD(uri: string) {
+        return this.helper.fetch(uri);
```

Comment:
It would be good to do some error handling here for TDs that cannot be fetched. See https://github.com/tum-esi/wade/issues/40